crates/biome_diagnostics/src/diagnostic.rs

```diff
@@ -163,38 +164,77 @@ impl Display for Severity {
 
 /// Internal enum used to automatically generate bit offsets for [DiagnosticTags]
 /// and help with the implementation of `serde` and `schemars` for tags.
-#[derive(Serialize, Deserialize)]
+#[derive(Debug, Copy, Clone, Serialize, Deserialize)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase")]
+#[bitflags]
+#[repr(u8)]
 pub(super) enum DiagnosticTag {
-    Fixable,
-    Internal,
-    UnnecessaryCode,
-    DeprecatedCode,
-    Verbose,
+    Fixable = 1 << 0,
+    Internal = 1 << 1,
+    UnnecessaryCode = 1 << 2,
+    DeprecatedCode = 1 << 3,
+    Verbose = 1 << 4,
 }
 
-bitflags! {
-    #[derive(Default, Debug, Copy, Clone, Eq, PartialEq)]
-    pub struct DiagnosticTags: u8 {
-        /// This diagnostic has a fix suggestion.
-        const FIXABLE = 1 << DiagnosticTag::Fixable as u8;
-        /// This diagnostic results from an internal error.
-        const INTERNAL = 1 << DiagnosticTag::Internal as u8;
-        /// This diagnostic tags unused or unnecessary code, this may change
-        /// how the diagnostic is render in editors.
-        const UNNECESSARY_CODE = 1 << DiagnosticTag::UnnecessaryCode as u8;
-        /// This diagnostic tags deprecated or obsolete code, this may change
-        /// how the diagnostic is render in editors.
-        const DEPRECATED_CODE = 1 << DiagnosticTag::DeprecatedCode as u8;
-        /// This diagnostic is verbose and should be printed only if the `--verbose` option is provided
-        const VERBOSE = 1 << DiagnosticTag::Verbose as u8;
+impl std::fmt::Display for DiagnosticTag {
```

Comment:
did we have a Display impl for this type before?

Reply:
No, sorry. I took another refactor as reference and I thought it would be a possible good convenience impl. Should I remove it?

Reply:
It’s unnecessary if we already derive `Debug` right?

Reply:
Yeah lets go ahead and remove it.

Reply:
done 😄